recipes/spfft/build.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX="${PREFIX}"  -DSPFFT_MPI=ON -DSPFFT_OMP=ON -DSPFFT_GPU_BACKEND=OFF -DSPFFT_GPU_DIRECT=OFF -DSPFFT_SINGLE_PRECISION=ON -DSPFFT_STATIC=OFF -DSPFFT_FORTRAN=ON -DSPFFT_BUILD_TESTS=OFF
+make -j2 install
```

Comment:
Is there a `make check`, `ctest` or `make test` that we can run here?

Reply:
There are tests that can be run, which I've added now.